tests.py

```diff
@@ -216,6 +216,60 @@ def test_create_or_update_create_time_catch_request_error(self, m):
                                                                "time",
                                                                "times"))
 
+    def test_create_or_update_time_with_junk_string_duration(self):
```

Comment:
So this should be something along the lines of this? https://github.com/osuosl/pymesync/blob/develop/tests.py#L214